src/app/modules/main/chat_section/controller.nim

```diff
@@ -58,6 +60,12 @@ proc delete*(self: Controller) =
 proc getActiveChatId*(self: Controller): string =
   return self.activeItemId
 
+proc getIsCurrentSectionActive*(self: Controller): bool =
```

Comment:
Hmm, can't we just return self.activeItemId == self.sectionId and drop setIsCurrentSectionActive() ?
Or maybe activeItemId does not keep current active item ?

Reply:
`activeItemId` is the ID of the active Item in **this** section, but before, the controller didn't know if the section it is in is the active section. So, what happened is that every section (communities and personal) would set their own activeChat as enabled, creating useless calls.